packages/plugins/connections/connection-mongodb/src/connections/MongoDBCollection/serialize.js

```diff
@@ -14,13 +14,15 @@
   limitations under the License.
 */
 
-import { ObjectID } from 'mongodb';
```

Comment:
Same error here.
`SyntaxError: Named export 'ObjectID' not found. The requested module 'mongodb' is a CommonJS module, which may not support all module.exports as named exports.`
Also 'ObjectID' was flagged as deprecated.